environment-osx-arm64.yml

```
@@ -10,7 +10,7 @@ dependencies:
   - python=>3.10
   - compilers
   - numpy>=1.17.0,<2
-  - scipy>=0.14,<1.14.0
+  - scipy>=0.14.0
```

Can probably pin to the current major version. No way 0.x would still work anyway?

```suggestion
  - scipy>=1,<2
```